### PR TITLE
DOC: change picks description in ICA.plot_properties()

### LINKED ISSUE #10264: DOC: Description of picks in `ica.plot_properties` is incorrect
With the `picks` keyword in `ica.plot_properties`, one can specify what ica sources to plot the properties of. So picks should code the ica source. However, at the moment, the description of picks is the [standard phrase](https://mne.tools/stable/generated/mne.preprocessing.ICA.html#mne.preprocessing.ICA.plot_properties): 

> Channels to include. Slices and lists of integers will be interpreted as channel indices. In lists, channel type strings (e.g., ['meg', 'eeg']) will pick channels of those types, channel name strings (e.g., ['MEG0111', 'MEG2623'] will pick the given channels. Can also be the string values “all” to pick all channels, or “data” to pick data channels. 

plus something specific to this plot: 

> None (default) will pick the first five sources. If more than one components were chosen in the picks, each one will be plotted in a separate figure.

#### Proposed documentation ehancement
The first part does not make sense here, and should be replaced by something like: 

> Sources to plot. None (default) will pick the first five sources. If more than one components were chosen in the picks, each one will be plotted in a separate figure.

### PR DESCRIPTION
**Pull request by eort**
Fixes #10264.

Like discussed, I updated the description what info picks represent.

### CODE REVIEW

**Comment by eort** [comment]
Not sure whether there is some pre-made pick information that can be applied to sources. So, I just oriented myself at the picks description in utils/docs.py

**Comment by mmagnuski** [comment]
small nitpick: I think to maximize clarity its better to refer to "components", not "sources".

File: mne/viz/ica.py [hunk 1]
```diff
@@ -272,7 +272,9 @@ def plot_ica_properties(ica, inst, picks=None, axes=None, dB=True,
         The ICA solution.
     inst : instance of Epochs or Raw
         The data to use in plotting properties.
-    %(picks_base)s the first five sources.
+    picks : str | list | slice | None
+        Sources to include. Slices and lists of integers will be interpreted
```
Comment (agramfort) [inline mne/viz/ica.py:276]:
```suggestion
        Components to include. Slices and lists of integers will be interpreted
```

File: mne/viz/ica.py [hunk 2]
```diff
@@ -272,7 +272,9 @@ def plot_ica_properties(ica, inst, picks=None, axes=None, dB=True,
...
+        as source indices. None (default) will use the first five sources.
```
Comment (agramfort) [inline mne/viz/ica.py:277]:
```suggestion
        as component indices. None (default) will use the first five components.
```

**Review by agramfort — APPROVED** [review]

File: mne/viz/ica.py [hunk 3]
```diff
@@ -272,8 +272,10 @@ def plot_ica_properties(ica, inst, picks=None, axes=None, dB=True,
         The ICA solution.
     inst : instance of Epochs or Raw
         The data to use in plotting properties.
-    %(picks_base)s the first five sources.
-        If more than one components were chosen in the picks,
+    picks : str | list | slice | None
+        Components to include. Slices and lists of integers will be interpreted
+        as component indices. None (default) will use the first five
+        components. If more than one components were chosen in the picks,
         each one will be plotted in a separate figure.
```
Comment (hoechenberger) [inline mne/viz/ica.py:279]:
```suggestion
```suggestion
        Components to include. Slices and lists of integers will be interpreted
        as component indices. ``None`` (default) will use the first five
        components. Each component will be plotted in a separate figure.
```

**Review by hoechenberger — COMMENTED** [review]
I made a small suggestion to make the docstring a little bit more concise, WDYT, @eort?

File: mne/viz/ica.py (hunk 3 above)
Comment (eort) [reply mne/viz/ica.py:279]: Looks good to me!

**Review by hoechenberger — APPROVED** [review]

**Pull request merged**